wiki/Interface/en.md

```diff
@@ -291,20 +291,24 @@ Below are the grade screens of the other game modes.
 
-1. You can change the name to record for this replay here.
-2. Your player bar. It shows your [PP](/wiki/Performance_Points), Global Rank, Total Score, Overall [Accuracy](/wiki/Accuracy), and level bar.
-3. Your options regarding the Beatmap.
-   1. Export Replay as `.osr`: Save the replay that can be used online.
-   2. Online Favourite: Make the Beatmap as your Online Favourite. It will show on your online profile under Beatmaps->Favourite Beatmaps
-4. Here is the difference made to your profile. This shows how far you've moved up or down the Global Ranking, how much was added to your Total Score, and how much your Overall Accuracy changed. The text fades out rather quickly, so don't miss it!
-5. Self-explanatory. Will not appear if offline, or if the beatmap is not uploaded.
-   1. Map Rank: Your Ranking on the map. The ranking includes replays that featured mods, even if you didn't use any yourself.
-   2. Overall: Your profile standing on the gamemode you just played, not just the beatmap.
-   3. Accuracy: Will only be counted when your old score is surpassed. Accuracy is the average of all passed beatmaps.
-   4. Ranked Score: Your Ranked Score. If it's an approved song, it will not increase at all.
-   5. Total Score: Grand total score. This will always increase even if you fail or don't surpass a high score in a beatmap. This counts towards your level.
-   6. To Next Rank: How much score you need to move your current map ranking position up.
-6. Same as #4, but it is reserved for monthly [Ranking Charts](https://osu.ppy.sh/rankings/osu/charts)
+1. Your player bar. It shows your [PP](/wiki/Performance_Points), Global Rank, Total Score, Overall [Accuracy](/wiki/Accuracy), and level bar.
+2. Save replay to Replays folder: That replay can be used offline.
+3. Online Favourite: Make the beatmap as your favourite. This will be visible in your osu! page, section "Beatmaps".
+4. Local Leaderboard: All your results are stored on the computer. To go to them, enter the [song selection screen](#song-selection-screen), you should see the [rankings](#rankings). Click on the drop-down menu and select "Local Rankings".
+5. Beatmap Ranking section. Will not appear if offline, or if the beatmap is not uploaded.
+   1. Overall: Your Ranking on the map. The result includes replays that featured mods, even if you didn't use any yourself.
+   2. Accuracy: How [precisely](/wiki/Accuracy) did you play the beatmap. Will only be counted when your old score is surpassed.
+   3. Max Combo: Your biggest combo on the map you played. If it's an unranked song, it will not increase at all.   
+   4. Ranked Score: Your [best result](/wiki/Score#ranked-score) on map. If it's an unranked song, it will not increase at all.
+   5. Total Score: Not taken into account, since it does not take a fate.
+   6. Performance: The received number of PP on map you played. If it's an unranked (or an approved) song, it will not increase at all.
```

Comment:
```suggestion
5. `Beatmap Ranking` section. Available only for maps with online leaderboards ([qualified](wiki/Beatmaps/en.md#qualified), [ranked](wiki/Beatmaps/en.md#ranked), or [loved](wiki/Beatmaps/en.md#loved)).
   1. `Overall`: Your position on the map's leaderboard, where you compete against players that used [mods](/wiki/Game_modifier/en.md), even if you didn't use any yourself.
   2. `Accuracy`: How [precisely](/wiki/Accuracy) did you play the beatmap. Will only be counted when your old score is surpassed.
   3. `Max Combo`: Your longest combo on the map you played.
   4. `Ranked Score`: Your [best result](/wiki/Score#ranked-score) on the map.
   5. `Total Score`: Not taken into account, since it does not affect your position in online rankings.
   6. `Performance`: The amount of [unweighted PP](/wiki/Performance_points/en.md#why-didnt-i-gain-the-full-amount-of-pp-from-a-map-i-played) you would receive for the play.
```